corehq/messaging/scheduling/migrations/0024_app_id.py

```diff
@@ -0,0 +1,57 @@
+from django.db import migrations, models
+from itertools import chain
+
+from corehq.apps.app_manager.dbaccessors import get_apps_in_domain
+from corehq.messaging.scheduling.models import AlertSchedule, TimedSchedule
+from corehq.util.django_migrations import skip_on_fresh_install
+
+
+def _update_model(model, domain, domain_forms):
+    if model.form_unique_id is None:
+        return None
+
+    if domain not in domain_forms:
+        domain_forms[domain] = {}
+        apps = get_apps_in_domain(domain)
+        for app in apps:
+            for module in app.modules:
+                for form in module.get_forms():
+                    domain_forms[domain][form.unique_id] = app.get_id
+
+    model.app_id = domain_forms[domain].get(model.form_unique_id, None)
+    if model.app_id:
+        model.save()
```

Comment:
should this be using `.update` as well?

Reply:
I was going to be lazy and leave this one as-is, didn't think ~1000 updates was a lot.

Reply:
That sounds reasonable